pyaedt/application/Analysis3D.py

```diff
@@ -1277,6 +1277,117 @@ def import_dxf(
         self.oeditor.ImportDXF(vArg1)
         return True
 
+    @pyaedt_function_handler
+    def import_gds_3d(self, gds_file_path, gds_number_list, elevation_list=None, thickness_list=None, import_method=1):
```

Comment:
@amichel0205 for me it's better to have gds_numbers rather than gds_number_list, elevation rather than elevation_list and thickness rather than thickness_list.
There's no need to specify list in the name because you are already explain it in the docstring below.

Reply:
I've implemented it. I run test before to commit and push

Reply:
@amichel0205 it's not what I meant.
```suggestion
    def import_gds_3d(self, gds_file, gds_number, elevation=None, thickness=None, import_method=1):
```

Reply:
done